wct.conf.json

```diff
@@ -0,0 +1,20 @@
+{
+  "verbose": false,
+  "trackConsoleError": false,
+  "plugins": {
+    "local": {
+      "browsers": ["chrome"]
+    },
+    "sauce": {
+      "disabled": false,
+      "browsers": [{
+        "browserName": "firefox",
```

Comment:
I think we should test in Edge as well. Plus we should be able to test firefox in `"local"`